infra/event-sync/src/main/java/com/simprints/infra/eventsync/sync/down/workers/EventDownSyncDownloaderWorker.kt

```diff
@@ -117,3 +136,12 @@ internal suspend fun WorkInfo.extractDownSyncProgress(eventSyncCache: EventSyncC
     val cached = eventSyncCache.readProgress(id.toString())
     return maxOf(progress, output, cached)
 }
+
+internal suspend fun WorkInfo.extractDownSyncMaxCount(eventSyncCache: EventSyncCache): Int {
+    val progress = this.progress.getInt(PROGRESS_DOWN_MAX_SYNC, -1)
+    val output = this.outputData.getInt(OUTPUT_DOWN_MAX_SYNC, -1)
```

Comment:
Consts can probably be named a little more clearly - PROGRESS_DOWN_SYNC & MAX_DOWN_SYNC or something similar.

Reply:
I would say that it is a matter of preference. I wanted to keep the prefix as similar as possible to the non-max constants. It would also imply that similar constant should be used together all the time.
Since those constants are only relevant in the context of this file, I will leave it as is for now.